src/components/Table/GridTable.tsx

```diff
@@ -959,7 +966,7 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
           .filter((style) => style)
       : undefined;
 
-  const rowStyleCellCss = maybeApplyFunction(row, rowStyle?.cellCss);
+  const rowStyleCellCss = maybeApplyFunction(row as any, rowStyle?.cellCss);
```

Comment:
The `IfAny` I did to "fix" `GridDataRow<any>` broke these two other spots, just doing an `as any` on them for now.